### ISSUE TITLE
Saldo oppdaterer seg ikke innenfor en session

### ISSUE THREAD

**Issue by andreasoye** [Sbanken/api-examples#7]
Når en kjører kall for å overføre penger så oppdaterer transaksjonslisten seg men ikke saldo.
Dette ser ut til å være pga. caching av saldo innenfor en session. Bør oppdatere dokumentasjonen tilsvarende (evt. fikse bug).

**Comment by sbankenOB** [comment]
Transfer operations performed in the internet bank will not invalidate the API cache for accounts. We are looking into possibilities for shared caching.

**Comment by heumn** [comment]
This is a collection of calls only asking for a particular account`s `available` balance

```
Received available is: 2291.23
Received available is: 2291.23
Received available is: 1791.23
Received available is: 2291.23
Received available is: 2291.23
Received available is: 2291.23
Received available is: 2291.23
Received available is: 1791.23
Received available is: 2291.23
Received available is: 2291.23
Received available is: 2291.23
Received available is: 2291.23
```

How did I manage to get in this situation? Made a lot of transfers between two accounts (back and forth) and for the next 15 minutes the cached balance has been inconsistent without me making any new calls to the `Transfers` endpoint.

It seems like you have some caching invalidation issues
